### PR TITLE
Nma 1337

### PR DESCRIPTION
**Pull request by hadia**
## Issue being fixed or feature implemented

- NMA-1337 User should be able to enter amount in their default fiat currency 
- Fix issue to Min amount  value  message warning to show default fiat currency
- NMA-1333 don't receive dash in the wallet when Convert Crypto( USDC -> Dash wallet)
- The ConvertCrypto and transfer to coinbase confirmation is showing the wrong direction message 
- The retry button that is on the failed transfer state is supposed to attempt to try to transfer the Dash to the DW

## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

File: integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/ConvertViewViewModel.kt [hunk 1]
```diff
@@ -142,8 +142,13 @@ class ConvertViewViewModel @Inject constructor(
         }
         this._selectedCryptoCurrencyAccount.value = account
 
+        //To check if the user has different fiat than usd the min is 2 usd
+        val minFaitValue = "2".toBigDecimal() / account.currencyToUSDExchangeRate.toBigDecimal()
```
Comment (Syn-McJ) [inline integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/ConvertViewViewModel.kt:146]: What if the min fiat value changes?

**Review by Syn-McJ — APPROVED** [review]
Looks good but I have some concerns about the hardcoded "min fiat" value. Doesn't seem like a good solution.

File: integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/ConvertViewViewModel.kt (hunk 1 above)
Comment (hadia) [reply integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/ConvertViewViewModel.kt:146]: this is an assumption from our side I mean value is 2 USD

Comment (HashEngineering) [reply integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/ConvertViewViewModel.kt:146]: we can address this in the next coinbase related story.

**Review by HashEngineering — APPROVED** [review]

**Pull request merged**